config/domains.txt

```diff
@@ -20574,3 +20574,93 @@ uintacounty.com
 wrightwyoming.com
 wyomingbusiness.org
 wyomingtourism.org
+
+// Brazil Federal Government
+eb.mil.br
```

Comment:
`.gov.br` and `.mil.br` are already listed and would include any subdomains. I believe if you add `def.br` and `mp.br`, this entire list should be included.

Reply:
So I just need to add, `gov.br` and it will actually add all subdomains that terminate with `gov.br.`?